submarine-workbench/workbench-web-ng/src/app/pages/user/register/register.component.ts

```diff
@@ -18,14 +18,86 @@
  */
 
 import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { Router } from '@angular/router';
 
 @Component({
   selector: 'submarine-register',
   templateUrl: './register.component.html',
   styleUrls: ['./register.component.scss']
 })
 export class RegisterComponent implements OnInit {
-  constructor() {}
+  validateForm: FormGroup;
+  // mock data: must be removed in the future
```

Comment:
You can remind yourself by adding `TODO(kevin85421): must be removed in the future`, all TODO will be displayed in the `TODO view` in IDEA.

Reply:
Thank for your review, @liuxunorg 
I have fixed the problem.